content/en/tracing/setup/java.md

```diff
@@ -199,11 +199,11 @@ The tracer is configured using System Properties and Environment Variables as fo
+| `dd.tags`                              | `DD_TAGS`                              | `null`                            | (Example: `key1:value1,key2:value2`) A list of default tags to be added to every span, profile, and JMX metric.                                                                                                                                                       |
 | `dd.writer.type`                       | `DD_WRITER_TYPE`                       | `DDAgentWriter`                   | Default value sends traces to the Agent. Configuring with `LoggingWriter` instead writes traces out to the console.                                                                                                                                                   |
 | `dd.agent.host`                        | `DD_AGENT_HOST`                        | `localhost`                       | Hostname for where to send traces to. If using a containerized environment, configure this to be the host IP. See [Tracing Docker Applications][1] for more details.                                                                                                  |
 | `dd.trace.agent.port`                  | `DD_TRACE_AGENT_PORT`                  | `8126`                            | Port number the Agent is listening on for configured host.                                                                                                                                                                                                            |
 | `dd.trace.agent.unix.domain.socket`    | `DD_TRACE_AGENT_UNIX_DOMAIN_SOCKET`    | `null`                            | This can be used to direct trace traffic to a proxy, to later be sent to a remote Datadog Agent.                                                                                                                                                                      |
-| `dd.trace.global.tags`                 | `DD_TRACE_GLOBAL_TAGS`                 | `null`                            | (Example: `key1:value1,key2:value2`) A list of default tags to be added to every span and every JMX metric. This value is merged into `trace.span.tags` and `trace.jmx.tags` to provide single place to configure both.                                               |
 | `dd.trace.span.tags`                   | `DD_TRACE_SPAN_TAGS`                   | `null`                            | (Example: `key1:value1,key2:value2`) A list of default tags to be added to every span. Tags of the same name added directly to a span overwrite the defaults provided here.                                                                                           |
```

Comment:
Probably we want only to document `DD_TAGS` as the way to set the same tags across products. Thoughts?

Reply:
I'm ok with that.  I'll remove the others.